index.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
```

Comment:
There are lots of errors here that block me from running your Mocha tests (and this is after I made the initial correction in your test scripts).